runhouse/constants.py

```diff
@@ -62,3 +63,11 @@
 # cluster status constants
 DOUBLE_SPACE_UNICODE = "\u00A0\u00A0"
 BULLET_UNICODE = "\u2022"
+STATUS_CHECK_OFF = -1
```

Comment:
can we just reuse `DEFAULT_STATUS_CHECK_INTERVAL` instead of maintaining a separate variable?

Reply:
thought about it, but `DEFAULT_STATUS_CHECK_INTERVAL` can be also a positive integer (if in the future we'll decide to turn on pinging to den by default) and in that case we would not like to return.  @jlewitt1